swift/apple/FirezoneKit/Sources/FirezoneKit/Views/GrantVPNView.swift

```diff
@@ -17,7 +17,13 @@ final class GrantVPNViewModel: ObservableObject {
 
   func grantPermissionButtonTapped() {
     Log.app.log("\(#function)")
-    store.createVPNProfile()
+    Task {
+      do {
+        try await store.createVPNProfile()
+      } catch {
+        Log.app.error("\(#function): \(error)")
```

Comment:
We were swallowing these silently before. All of these logger calls will be improved with #7560 and #7004